src/expint.jl

```diff
@@ -214,16 +214,17 @@ function En_cf_gamma(ν::Number, z::Number, n::Int=1000)
     ϵ = 10*eps(real(B))
     scale = sqrt(floatmax(real(A)))
 
-    iters = 1
-    j = 1
-    for i = 2:n
+    iters = 0
+    for i = 1:n
         iters += 1
 
-        term = iseven(i) ? -(i÷2 - 1 - ν)*z : z*(i÷2)
-        A, Aprev = (i - ν)*A + term * Aprev, A
-        B, Bprev = (i - ν)*B + term * Bprev, B
+        a = iseven(i) ? (i÷2)*z : -((i + 1)÷2 - ν)*z
+        b = (i + 1 - ν)
 
-        conv = abs(Aprev*B - A*Bprev) < ϵ*abs(B*Bprev)
+        A, Aprev = b*A + a*Aprev, A
+        B, Bprev = b*B + a*Bprev, B
+
+        conv = abs(Aprev*B - A*Bprev) < ϵ*abs(Aprev*Bprev)
```

Comment:
What's the reason for not using
```suggestion
        conv = abs(Aprev*B - A*Bprev) < ϵ*abs(Aprev*Bprev)
```
here? I thought one would want to check `|B/A - Bprev/Aprev| < eps`, which would be equivalent to `|Aprev*B - A*Bprev| < eps*|A*Aprev|` (instead of `... < eps*|Aprev*Bprev|`).

Reply:
My version is testing the relative change instead of the absolute change. I'd think that the relative change would be the appropriate test.

Reply:
Yes, I agree, probably the relative change is more appropriate. But in this case, wouldn't `|(B/A - Bprev/Aprev)/(Bprev/Aprev)| < eps` (which I assume one would use?) be equivalent with `|B*Aprev - Bprev*A| < eps*|Bprev*A|` and `|(B/A - Bprev/Aprev)/(B/A)| < eps` equivalent with `|B*Aprev - Bprev*A| < eps * |B*Aprev|`?

Reply:
You are right. I've updated the PR.